Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1315,14 +1308,13 @@ bool TextureCacheBase::LoadTextureFromOverlappingTextures(TCacheEntry* entry_to_
   // instead, which would reduce the amount of copying work here.
   std::vector<TCacheEntry*> candidates;
 
-  auto iter = FindOverlappingTextures(entry_to_update->addr, entry_to_update->size_in_bytes);
+  auto iter = FindOverlappingTextures(tex_info.address, tex_info.total_bytes);
   while (iter.first != iter.second)
   {
     TCacheEntry* entry = iter.first->second;
-    if (entry != entry_to_update && entry->IsCopy() && !entry->tmem_only &&
-        entry->references.count(entry_to_update) == 0 &&
```

Comment:
Was this intentionally removed?  I haven't been following the code base, maybe this is to match the other function?

Reply:
With entry_to_update always being a new texture, the existing textures shouldn't have any references to it.